### PR TITLE
Hdfs hostname port

### PR DESCRIPTION
**Pull request by kerkinos**

### CODE REVIEW

File: src/main/java/io/hops/hdfs/AbstractFacade.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+package io.hops.hdfs;
+
+import java.util.List;
+import javax.persistence.EntityManager;
+
+/**
+ *
+ * @author kerkinos
+ */
+public abstract class AbstractFacade<T> {
```
Comment (jimdowling) [inline src/main/java/io/hops/hdfs/AbstractFacade.java]: We already have a class called AbstractFacade.java. Can you use that one instead?

File: src/main/java/io/hops/hdfs/EndpointService.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    }
+
+    @EJB
+    private HdfsLeDescriptorsFacade hdfsLeDescriptorsFacade;
+    @EJB
+    private NoCacheResponse noCacheResponse;
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response findEndpoint(
```
Comment (jimdowling) [inline src/main/java/io/hops/hdfs/EndpointService.java]: We need to add security for this, check it's a valid user who has logged in.   Add:

  @AllowedRoles(roles = {AllowedRoles.ALL})

**Comment by jimdowling** [comment]
If you make those two fixes, you can merge this.

**Comment by kerkinos** [comment]
Done and done

**Pull request merged**